From 8c92b6f204ffa1f133803c756511f73a085fb512 Mon Sep 17 00:00:00 2001
From: Ofer Heifetz <oferh@marvell.com>
Date: Tue, 14 Nov 2017 10:04:02 +0200
Subject: [PATCH 1271/1345] crypto: inside-secure: adjust firmware path

commit  19f998bcbe1a9007c20c297c0201cba84ad95217 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

New ap810 uses new eip197d which needs different firmware then
the cp110 eip197b, to differentiate between the two firmwares,
a new directory was added to the firmware path.

Change-Id: I9b3e149c15768c7a33b293294dca0dabe59a1508
Signed-off-by: Ofer Heifetz <oferh@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/46293
Reviewed-by: Igal Liberman <igall@marvell.com>
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/51652
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/crypto/inside-secure/safexcel.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/crypto/inside-secure/safexcel.c b/drivers/crypto/inside-secure/safexcel.c
index 08892ca..302cfda 100644
--- a/drivers/crypto/inside-secure/safexcel.c
+++ b/drivers/crypto/inside-secure/safexcel.c
@@ -255,8 +255,8 @@ static int eip197_load_fw(struct device *dev, struct safexcel_crypto_priv *priv)
 	const u32		*fw_data;
 	int			i, ret;
 	u32			fw_size, reg;
-	const char		*fw_name[MAX_FW_NR] = {"eip197/ifpp.bin",
-						       "eip197/ipue.bin"};
+	const char		*fw_name[MAX_FW_NR] = {"eip197/197b/ifpp.bin",
+						       "eip197/197b/ipue.bin"};
 
 	for (i = 0; i < MAX_FW_NR; i++) {
 		ret = request_firmware(&fw[i], fw_name[i], dev);
-- 
1.7.9.5

